app/javascript/admin/controllers/sidebar_controller.js

```diff
@@ -2,14 +2,14 @@ import { Controller } from 'stimulus';
 
 // eslint-disable-next-line no-restricted-syntax
 export default class SidebarController extends Controller {
-  static targets = [
-    'submenu'
-  ];
+  static targets = ['submenu'];
```

Comment:
There's a few changes like this one added by the pre-commit hooks. I can get rid of them if someone feels like they shouldn't be included

Reply:
I don't feel strongly about it, but if it's simple enough to revert then I think it would help keep the scope of this PR clearer.